docs/src/pages/[platform]/components/heading/HeadingPropControls.tsx

```diff
@@ -17,9 +22,18 @@ interface HeadingPropControlsInterface {
 export const HeadingPropControls: HeadingPropControlsInterface = ({
   level,
   setLevel,
+  isTruncated,
+  setIsTruncated,
+  value,
+  setValue,
 }) => {
   return (
     <Flex direction="column">
+      <TextField
+        label="Displayed Text"
+        onChange={(event) => setValue(event.target.value)}
+        defaultValue={value}
```

Comment:
We probably want to use `value` instead of `defaultValue` coz it is used as controlled component now